### ISSUE TITLE
Code and details of re-evaluation of SNLI-VE in your paper? (Unable to reproduce your 73.02% Acc.)

### ISSUE THREAD

**Issue by HearyShen** [virginie-do/e-SNLI-VE#8]
Hi, 

In your paper, Section 2.2, **Re-evaluation of Visual-Textual Entailment**, you announced a **73.02%** accuracy of BUTD model on SNLI-VE dataset, which outperforms Xie et al's _68.90%_ by a big margin.

You didn't release the code and configs of this section, so I implemented this BUTD model with PyTorch according to your code and the original [coling18-gte](https://github.com/claudiogreco/coling18-gte):
- For object features, I used your image feature files, however, I achieved **68.27%**, similar to Xie et al's _68.90%_, significantly lower than your **73.02%**.
- For image feature map, I used ResNet101 and achieved **71.10%**, similar to Xie et al's _70.30%_, while you didn't give this accuracy.

In conclusion, my experiment results corroborate Xie et al's work while can not reproduce the 73.02% announced in your paper.

Would you please release your code and experiment details(maybe configs, logs or etc.) to help me reproduce your 73.02% accuracy?

Thanks!

Reference:
Xie N, Lai F, Doran D, et al. Visual entailment: A novel task for fine-grained image understanding[J]. arXiv preprint arXiv:1901.06706, 2019.
